crates/coordinate_systems/src/transform_nalgebra.rs

```diff
@@ -0,0 +1,85 @@
+use nalgebra::{AbstractRotation, Isometry, Isometry2, SimdRealField};
+
+use crate::{Isometry3, Orientation, Point, Pose, Transform, UnitComplex, Vector2};
+
+// Isometry
+
+impl<From, To, Scalar, Rotation, const DIMENSION: usize>
+    Transform<From, To, Isometry<Scalar, Rotation, DIMENSION>>
+where
+    Scalar::Element: SimdRealField,
+    Scalar: SimdRealField,
+    Rotation: AbstractRotation<Scalar, DIMENSION>,
+{
+    pub fn identity() -> Self {
+        Self::wrap(nalgebra::Isometry::identity())
+    }
+
+    pub fn inverse(&self) -> Transform<To, From, Isometry<Scalar, Rotation, DIMENSION>> {
+        Transform::<To, From, _>::wrap(self.inner.inverse())
+    }
+
+    // TODO: naming kloppen
```

Comment:
todo

Reply:
Do you have a suggestion?

Reply:
`ground_to_field.as_pose().position()`? maybe?

Reply:
Done